qiskit/pulse/utils.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Pulse utilities.
+"""
+import warnings
+
+from typing import List
+
+from .channels import AcquireChannel, MemorySlot
+from .commands import Acquire, AcquireInstruction
+from .interfaces import ScheduleComponent
+from .schedule import Schedule
+from .cmd_def import CmdDef
+
+
+def align_measures(schedule: ScheduleComponent, cmd_def: CmdDef) -> Schedule:
```

Comment:
We really want to align measures across a group of schedules comprising a single experiment so this should actually take in a list of schedules